include/openssl/ocsp.h

```diff
@@ -1,15 +1,5 @@
-/*
- * Copyright 2000-2020 The OpenSSL Project Authors. All Rights Reserved.
- *
- * Licensed under the Apache License 2.0 (the "License").  You may not use
- * this file except in compliance with the License.  You can obtain a copy
- * in the file LICENSE in the source distribution or at
- * https://www.openssl.org/source/license.html
- */
-
-//
-// Created by Chiang, Samuel on 4/22/21.
-//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this a file we are taking from OpenSSL, or something that we created? If it is from OpenSSL, I think we should still be using their copyright header.

Reply:
I was thinking if we should add our copyright header since the file was teared down significantly from the original OpenSSL file. But every function definition moving forward in this file is going to mirror OpenSSL for backwards compatibility, so I agree we should use their copyright header.